<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1665</width>
    <height>1125</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>811</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>登录</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>411</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>401</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radio_stu">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>学生</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_tea">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>教师</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>密码  </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textEdit_3">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string> 老师界面</string>
     </attribute>
     <widget class="QWidget" name="teawidget" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>431</width>
        <height>321</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>371</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="tealayer">
        <item>
         <widget class="QPushButton" name="course">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
          <property name="text">
           <string>查看全校课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="self">
          <property name="text">
           <string>查看自身课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showstu">
          <property name="text">
           <string>显示学生</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="input">
          <property name="text">
           <string>输入和查看成绩</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showgrade">
          <property name="text">
           <string>查看学生成绩</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string> 学生界面</string>
     </attribute>
     <widget class="QWidget" name="stuwidget" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>291</width>
        <height>421</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>281</width>
         <height>381</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="stulayer">
        <item>
         <widget class="QPushButton" name="s_course">
          <property name="text">
           <string>查看全校课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="s_show_own_course">
          <property name="text">
           <string>查看自身课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="choose_course">
          <property name="text">
           <string>选课</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="giveup">
          <property name="text">
           <string>退课</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="show_all_grade">
          <property name="text">
           <string>查看课程成绩</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>输入成绩</string>
     </attribute>
     <widget class="QRadioButton" name="radiocom">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>必修课</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioop">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>选修课</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>课编号</string>
      </property>
     </widget>
     <widget class="QWidget" name="inwid" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>341</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="conf">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>确认编号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confg">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>输入成绩</string>
      </property>
     </widget>
     <widget class="QPushButton" name="conhand">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>540</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>提交成绩</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>查看学生成绩</string>
     </attribute>
     <widget class="QWidget" name="teag" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>441</width>
        <height>541</height>
       </rect>
      </property>
      <widget class="QPushButton" name="t_upgrade">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>311</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>按成绩升序</string>
       </property>
      </widget>
      <widget class="QPushButton" name="t_downgrade">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>311</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>按成绩降序</string>
       </property>
      </widget>
      <widget class="QPushButton" name="t_upnum">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>260</y>
         <width>311</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>按学号升序</string>
       </property>
      </widget>
      <widget class="QPushButton" name="t_downgrade_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>390</y>
         <width>311</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>按学号降序</string>
       </property>
      </widget>
     </widget>
     <widget class="QRadioButton" name="com2">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>10</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>必修</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="op2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>选修</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>查看课程成绩</string>
     </attribute>
     <widget class="QWidget" name="stug" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>441</width>
        <height>501</height>
       </rect>
      </property>
      <widget class="QPushButton" name="s_upgrade">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>341</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>按成绩升序</string>
       </property>
      </widget>
      <widget class="QPushButton" name="s_downgrade">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>341</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>按成绩降序</string>
       </property>
      </widget>
      <widget class="QPushButton" name="s_upgpa">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>341</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>按绩点升序</string>
       </property>
      </widget>
      <widget class="QPushButton" name="s_downgpa">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>341</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>按绩点降序</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>860</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>登出</string>
    </property>
   </widget>
   <widget class="QLabel" name="data">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>691</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_2">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>750</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新 </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1665</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
